renderlib/command.cpp

```diff
@@ -31,7 +32,8 @@ LoadOmeTifCommand::execute(ExecutionContext* c)
 {
   LOG_WARNING << "LoadOmeTif command is deprecated. Prefer LoadVolumeFromFile command.";
   LOG_DEBUG << "LoadOmeTif command: " << m_data.m_name;
-  if (stat(m_data.m_name.c_str(), nullptr) == 0) {
```

Comment:
FMI, was stat encountering an error when called w/ nullptr from python but not in cpp? (and if so, do you know why)

Reply:
Good question.  I honestly don't know how I was able to load files in GUI but not Python (agave running in server mode).  I haven't tried to repro on any OS other than MacOS. At first I thought it might be a permission issue.  However, the error I was getting indicated that it couldn't handle passing a null buffer in as the second argument.  So I feel much more comfortable that I am calling stat() in a way that should work more universally.  

I'm going to continue with more testing both GUI and server modes on other platforms to make sure these basic functions are working.

Reply:
Oh wait, I do know the answer to the first part.  The stat calls were added as part of an effort to factor Qt out of the "back end".  The front end actually has a similar stat call but uses a higher level Qt function which continues to work when in GUI mode.  So when in gui mode, it doesn't use quite the same function to test file existence.

Reply:
https://github.com/allen-cell-animated/agave/blob/main/agave_app/agaveGui.cpp#L345